website/frontend/management/commands/scraper.py

```diff
@@ -356,8 +386,10 @@ def update_articles(todays_git_dir):
     logger.info('Got all %s urls; storing to database' % len(all_urls))
     for i, url in enumerate(all_urls):
         logger.debug('Woo: %d/%d is %s' % (i+1, len(all_urls), url))
+        # Looks like it skips URLs longer than 255?
         if len(url) > 255:  #Icky hack, but otherwise they're truncated in DB.
             continue
+        # Is there an index on this column?
```

Comment:
Yes, it follows from `db_index=True` in models.py.

Reply:
Yes, sorry.  It was not my intention to include all of these comments in this PR.  Some of them were just notes for me to follow up on later.